packages/tools/generator-pie-component/.gitignore

```diff
@@ -1 +1,2 @@
-generators
\ No newline at end of file
+generators
+package-lock.json
```

Comment:
I don't think we need to add this because we should be using `yarn`

Reply:
For context, @bntsv was having issues following the install instructions for the generator (More to do with node setup on dev machine, not our docs).

We ended up getting around the issue by building `generator-pie-component` and using `npm link` to use the local version of the package when running `yo @justeattakeaway/pie-component`.

Happy for this to be reverted and done in another ticket though

Reply:
If we can just remove this line I'll approve, we don't need to check the package lock file in